Cargo.toml

```diff
@@ -35,6 +35,7 @@ futures       = { version = "0.1", optional = true }
 inotify-sys   = "0.1.3"
 libc          = "0.2"
 mio           = { version = "0.6", optional = true }
+tokio         = { version = "0.1", optional = true }
```

Comment:
Is it possible to move that dependency to `[dev-dependencies]`, or will that cause problems, due to the dependency being optional?

Reply:
yes, it causes problems because of that